Renegade X Launcher/VersionCheck.cs

```diff
@@ -38,8 +38,8 @@ static VersionCheck()
         {
             _launcherVersion = new Version
             {
-                Name = "0.00",
-                Number = 00
+                Name = "0.82",
+                Number = 82

```

Comment:
Please revert this so we can keep the version at 0 on master; we'll add logic later to suppress the update prompt for version 0. Otherwise, we should only be changing the version number on the end release branches, which is something I'll handle.